config/src/test/java/org/hyperledger/besu/config/GenesisConfigFileTest.java

```diff
@@ -198,6 +198,26 @@ public void shouldGetEmptyTerminalTotalDifficultyAtGenesis() {
     assertThat(EMPTY_CONFIG.getConfigOptions().getTerminalTotalDifficulty()).isNotPresent();
   }
 
+  @Test
+  public void shouldFindParisForkAndAlias() {
+    GenesisConfigFile parisGenesis =
+        GenesisConfigFile.fromConfig("{\"config\":{\"parisBlock\":10},\"baseFeePerGas\":\"0xa\"}");
+    assertThat(parisGenesis.getForks().size()).isEqualTo(1);
+    assertThat(parisGenesis.getForks().get(0)).isEqualTo(10L);
+
+    GenesisConfigFile premergeForkGenesis =
+        GenesisConfigFile.fromConfig(
+            "{\"config\":{\"preMergeForkBlock\":11},\"baseFeePerGas\":\"0xa\"}");
+    assertThat(premergeForkGenesis.getForks().size()).isEqualTo(1);
+    assertThat(premergeForkGenesis.getForks().get(0)).isEqualTo(11L);
+
+    GenesisConfigFile parisOverAlias =
+        GenesisConfigFile.fromConfig(
+            "{\"config\":{\"parisBlock\":10,\"preMergeForkBlock\":11},\"baseFeePerGas\":\"0xa\"}");
```

Comment:
Change my mind: this should be an error not an override

Reply:
There are a number of Genesis files out for kiln and devnet5 that reference the prior fork name.  The alias is a way to ease the friction of the change.  

I am thinking we should drop it before we merge any of the long lived test nets, possibly before we release 22.1.3. But I would like to make the UX for testers a bit smoother in the interim.

Reply:
One or the other is fine.  We did this with berlin and eip1559.  But specifying both should be an error.  I you went in to add Paris you should also remove preMergeFork

Reply:
Ah I gotcha.  That makes sense.  I thought you were referring to just having an alias 👍